### DISCUSSION TITLE
Bug: Filiment Manager

### DISCUSSION THREAD

**Discussion by JesNoFear** [OctoFarm/OctoFarm#1305] [General]
when using the Filament Manager, going in to edit a spool's properties, the USED text filed is populated with remaining. But Saves to the USED field.
Its extremely annoying to edit a unrelated property to then have to recalculate / repopulate how much is used before saving.

**Comment by NotExpectedYet** [comment]
Think this one passed me by. 

It should be in issues. 

If it's not resolved let me know and I'll add it to the issues to resolve next chance I get.
